examples/chain_client/exchange/19_MsgLiquidatePosition.py

```diff
@@ -37,7 +37,7 @@ async def main() -> None:
     fee_recipient = "inj1hkhdaj2a2clmq5jq6mspsggqs32vynpk228q3r"
     cid = str(uuid.uuid4())
 
-    order = composer.derivative_order(
+    order = composer.create_v2_derivative_order(
```

Comment:
why in some places we have `v2` in the middle and in some as a suffix?

Reply:
It is just according to what sounds better in the method name